floyd/client/data.py

```diff
@@ -72,19 +73,34 @@ def create(self, data):
             return response.json().get("id")
 
     def get(self, id):
-        response = self.request("GET",
-                                "{}{}".format(self.url, id))
-        data_dict = response.json()
-        return Data.from_dict(data_dict)
+        try:
+            response = self.request("GET",
+                                    "{}{}".format(self.url, id))
+            data_dict = response.json()
+            return Data.from_dict(data_dict)
+        except FloydException as e:
+            floyd_logger.info("Data {}: ERROR! {}".format(id, e.message))
+            return None
 
     def get_all(self):
-        response = self.request("GET",
-                                self.url,
-                                params="module_type=data")
-        experiments_dict = response.json()
-        return [Data.from_dict(expt) for expt in experiments_dict]
+        try:
+            response = self.request("GET",
+                                    self.url,
+                                    params="module_type=data")
+            experiments_dict = response.json()
+            return [Data.from_dict(expt) for expt in experiments_dict]
+        except FloydException as e:
+            floyd_logger.info(
+               "Error while retrieving data: {}".format(e.message)
+            )
```

Comment:
Minor: This line is not too long, so can we keep this in one line? (similar to the other functions in this class)